core/test/config/config-helpers-test.js

```diff
@@ -424,7 +424,7 @@ describe('.resolveGathererToDefn', () => {
   it('throws but not for missing gatherer when it has a node dependency error', async () => {
     const resultPromise =
       resolveGathererToDefn('../fixtures/invalid-gatherers/require-error.js', [], moduleDir);
-    await expect(resultPromise).rejects.toThrow(/Cannot find module/);
+    await expect(resultPromise).rejects.toThrow(/no such file or directory/);
```

Comment:
The "Cannot find module" text came from an error thrown in quibble. Updates in v0.7.0 of quibble appear to surface the actual node error instead. Good change IMO.